src/model.jl

```diff
@@ -807,10 +807,16 @@ julia> set_value(var, 1.3)
 """
 function JuMP.set_value(parameter::ParameterRef, new_value::Float64)
     p = parameter.model._parameters[parameter.index]
+    jm = parameter.model._jump_model
     if p isa ScalarParameter
         p.value = new_value
+        if jm!==nothing
+        
+        JuMP.set_parameter_value(jm[parameter.model._parameters[parameter.index].name],new_value)
+        end
```

Comment:
Aren't we looping over all parameters before every solve and are copying values over to the JuMP model in any case? Which then suggests we don't need to do this here?